public/tutorials/robust-smart-contracts-with-openzeppelin.md

```diff
@@ -112,51 +114,102 @@ Using less than 15 lines of hand-coded Solidity, we've created our own Ethereum
 
    The `import` statement within our `TutorialToken` contract will be automatically handled by the compiler, along with any subsequent imports within `StandardToken`.
 
-1. Now we can compile and migrate. In a second terminal window, run the TestRPC:
+1. Now we are ready to compile and migrate to the blockchain. We will do this using Truffle Develop, a development console that includes an in-memory blockchain, similar to the [EthereumJS TestRPC](https://github.com/ethereumjs/testrpc). In your terminal, make sure you are in the root of your project's directory and then launch Truffle Develop:
```

Comment:
"migrating to the blockchain" sounds weird in this context. Migrations deploy your contracts to the blockchain; deploying is the goal, and "running migrations" is the action (i.e., the means to an end). Here, I'd either use "deploy your contracts to the blockchain" or "run your migrations". Obviously the latter is confusing if you haven't introduced migrations yet.

Reply:
Ah I see. I confess I still get a little confused between "deploy" and "migrate" and was using them interchangeably in this case. Will update to say "deploy your contract to the blockchain" (there is only one, correct?).